index.html

```diff
@@ -15,26 +15,201 @@
+          what you see and have a project you need coded,
+          don’t hesitate to contact me.</p>
+        <button type="submit">Get my resume</button><img class="left-corner"
+          src="icons/illustration 2-about me mobile.svg" alt="">
+      <div class="containers">
+          <ul>
+            <li><img src="icons/Languages.svg" alt=""></li>
+            <li>
+              <h3>Languages</h3>
+            </li>
+            <li>
+              <ol>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ol>
+            </li>
+          </ul>
+        </div>
+        <div class="container2">
+          <div class="frameworks">
+            <ul>
+              <li><img src="icons/icon-frameworks.svg" alt=""></li>
+              <li>
+                <h3>Frameworks</h3>
+              </li>
+              <li>
+                <ol>
+                  <li>React.js</li>
+                  <li>Ruby on Rails</li>
+                  <li>RSpec</li>
+                  <li>Capybara</li>
+                  <li>Selenium</li>
+                </ol>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="container3">
+          <div class="skills">
+            <ul>
+              <li><img src="icons/icon-skills.svg" alt="skills"></li>
+              <li>
+                <h3>Skills</h3>
+              </li>
+              <li>
+                <ol>
+                  <li>Database Management</li>
+                  <li>Version Control</li>
+                  <li>CLI</li>
+                  <li>Web Development</li>
+                  <li>API Design</li>
+                </ol>
+              </li>
+            </ul>
+          </div> </div>
+          <div class="container4">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          </div>
+          <div class="container5">
+            <form action="/action_page.php">
+              <input type="text" id="fname" name="fullname" placeholder="Full name">
+              <input type="text" id="email" name="lastname" placeholder="Your last name..">
+              <textarea placeholder="Hey I would like to connect to ..."></textarea>
+              <button type="submit">Get in touch</button>
+            </form>
+          </div>
+          <div class="container6">
+            <h5>tsigedejen@gmail.com</h5>
+            <ul>
+              <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
+              <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img
+                    src="icons/Linkedin icon.svg"></i></a></li>
+              <li><a href=""><img src="icons/medium.svg"></i></a></li>
+              <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
+              <li><a href=""><img src="icons/angellist.svg"></a></li>
```

Comment:
[OPTIONAL] As I mentioned above :point_up_2: please stick to to the Figma design your social media icons are not same as Figma design so kindly fix this issues as well :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face:  

**Your Design**
![Screenshot from 2023-06-03 14-02-10](https://github.com/tsigewossne/portfolio/assets/45226848/90cc608d-7b5c-4e5f-a2ae-7003c396c5c6)

**Figma Design**
![Screenshot from 2023-06-03 14-07-11](https://github.com/tsigewossne/portfolio/assets/45226848/39e43a13-3efe-4c5e-ba04-feb79edb9873)